index.js

```diff
@@ -44,7 +44,7 @@ referrer\:":referrer"|\
 agent\:":user-agent"|\
 duration\::response-time"}',
         { skip: req => req.headers['user-agent'] == 'ELB-HealthChecker/2.0' }));
-    app.use(statsd({host: 'graphite.nypr.digital', requestKey: env+'.'+serviceName}));
+    app.use(statsd({host: 'graphite.nypr.digital', namespace: env+'.'+serviceName}));
```

Comment:
can you update this string to use template strings? `${env}.${serviceName}`

Reply:
done